servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/NettyPipelinedConnectionTest.java

```diff
@@ -295,12 +295,12 @@ void channelCloseErrorsPendingReadCancelsPendingWrite() {
 
         assertThat(readSubscriber.awaitOnError(), is(instanceOf(ClosedChannelException.class)));
         assertThat(readSubscriber2.awaitOnError(), is(instanceOf(ClosedChannelException.class)));
-        writePublisher1Subscription.awaitCancelledUninterruptibly();
+        writePublisher1Subscription.awaitCancelled();
         assertFalse(writePublisher2.isSubscribed());
     }
 
     @Test
-    void readCancelClosesConnectionThenWriteDoesNotSubscribe() {
+    void readCancelClosesConnectionThenWriteDoesNotSubscribe() throws InterruptedException {
```

Comment:
Not a problem, but because you will anyway address the previous comment, I will ask you to change `InterruptedException` to `Exception` here. For tests it's common that they may throw multiple checked exceptions. It's easier to use `Exception` instead of declaring each one independently. Makes tests easier to read.